test/Datadog.Trace.Tests/HttpHeadersPropagatorTests.cs

```diff
@@ -12,12 +14,14 @@ public void HttpRequestMessage_InjectExtract_Identity()
         {
             const int traceId = 9;
             const int spanId = 7;
-            var headers = new HttpRequestMessage().Headers;
-            var context = new SpanContext(traceId, spanId);
+            const SamplingPriority samplingPriority = SamplingPriority.UserKeep;
 
-            headers.Inject(context);
+            HttpRequestHeaders httpHeaders = new HttpRequestMessage().Headers;
+            var context = new SpanContext(traceId, spanId, samplingPriority);
 
-            var resultContext = headers.Extract();
+            IHeadersCollection headersWrapper = HeadersFactory.Wrap(httpHeaders);
+            headersWrapper.InjectSpanContext(context);
+            var resultContext = headersWrapper.ExtractSpanContext();
 
             Assert.Equal(context.SpanId, resultContext.SpanId);
             Assert.Equal(context.TraceId, resultContext.TraceId);
```

Comment:
Should we assert sampling priority here?

Reply:
Definitely. 4d8fc3a7d6c667da60c5ea64797b364302f2a935